### DISCUSSION TITLE
Exclude certain referers from statistics

### DISCUSSION THREAD

**Discussion by Evaldash** [ibericode/koko-analytics#244] [Ideas]
My payment provider (Paysera) appears in the Referrers list. I would love to have the option to select which referers to ignore, and not include in the statistics.

**Comment by dannyvankooten** [comment]
@Evaldash Great idea! 

This is currently possible (alhough not super simple) by adding a filter on the `koko_analytics_referrer_blocklist` filter hook.

```php

add_filter('koko_analytics_referrer_blocklist', function($blocklist) {
    $blocklist[] = 'paysera.com';
    return $blocklist;
});
```
